backend/tests/services/object_storage/test_upload_grant_service.py

```
@@ -0,0 +1,197 @@
+    )
+
+
+def test_create_upload_grant_rejects_invalid_skill_identity(
+    test_db: Session,
+    test_user: User,
+):
+    with pytest.raises(InvalidSkillIdentityError):
+        object_storage_upload_grant_service.create_upload_grant(
+            db=test_db,
+            auth_context=AuthContext(user=test_user),
+            skill_identity_token="invalid-token",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "skill_identity_token"

(S106)